classes/People.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * People Class
+ *
+ * @package TheyWorkForYou
+ */
+
+namespace MySociety\TheyWorkForYou;
+
+include_once( dirname(__FILE__) . '/../www/includes/easyparliament/people.php' );
+
+/**
+ * People
+ */
+
+class People extends \PEOPLE {
+
+    public $type;
+    public $rep_name;
+    public $rep_plural;
+    public $house;
+    public $cons_type;
+    public $db;
+
+    public function __construct( $type ) {
+        global $this_page;
+        $this_page = $type;
+        $this->type = $type;
+        $this->db = new \ParlDB();
+        $this->setTypeData();
+    }
+
+    public function getData($args = array()) {
+        $data = $this->display($this->type, $args, 'none');
+
+        $user = new User();
+        if ( $this->type == 'mlas' ) {
```

Comment:
Do we want to show the regional MSPs too? Guess if we did might not want to be so prominent.